src/lib/containers/works_for_you.tsx

```diff
@@ -0,0 +1,163 @@
+import * as React from "react"
+import { ListView, ListViewDataSource, ScrollView, StyleSheet, View } from "react-native"
+import * as Relay from "react-relay"
+
+import Headline from "../components/text/headline"
+import SerifText from "../components/text/serif"
+import Notification from "../components/works_for_you/notification"
+import { LayoutEvent } from "../system/events"
+
+import colors from "../../data/colors"
+
+interface Props {
```

Comment:
Same comment here, you should extend the `RelayProps` interface rather than writing the Relay props by hand.